appveyor.yml

```diff
@@ -0,0 +1,25 @@
+platform:
+  - Win32
+  - x64
+
+configuration:
+  - Debug
+  - Release
+
+environment:
+  LIBGDIPLUS_VERSION_PREFIX: 4.2
+
+before_build:
+  - cmd: choco install -y wget
+  - cmd: wget "https://dl.hexchat.net/gtk-win32/vc14/x86/gtk-Win32.7z" -O gtk-Win32.7z
+  - cmd: 7z x gtk-Win32.7z -ogtk
+  - cmd: wget "https://dl.hexchat.net/gtk-win32/vc14/x64/gtk-x64.7z" -O gtk-x64.7z
+  - cmd: 7z x gtk-x64.7z -ogtk
```

Comment:
We don't need GTK. GTK shares dependencies with libgdiplus (cairo, fontconfig, libpng,...), and gtk-win32 provides read-to-use win32 distribution of those dependencies.

If there's another source we can pull those dependencies from, happy to hear. Normally I try to use CoApp packages on NuGet but they are either really old or absent.